src/werkzeug/_internal.py

```diff
@@ -239,7 +267,7 @@ def parse(args, kwargs):
     return parse
 
 
-def _date_to_unix(arg):
+def _date_to_unix(arg: datetime) -> int:
```

Comment:
The docs say this can take a tuple (specifically a timetuple), integer, or datetime, but monkeytype only picked up the datetime.

I know monkeytype is using runtime analysis, but this could be hindered by < 100% test coverage (or maybe just wasn't picked up?). Adding the type here makes it harder to identify that the typing for it is incomplete later.

Please review the types that were generated to make sure there's no obvious issues.

Reply:
Yeah – I'm 100% going over and checking these types. They're not 100% there; it looks like it's just inserting whatever types it found the variables/parameters/whatever the type theory term is to be when running the tests so some manual review and pruning will be needed.
I'll probably also define some aliases to make things clearer/neater and a lot of `# type: ignore` comments in the meantime.